ca/ca.go

```diff
@@ -113,27 +113,29 @@ func (ca *CA) Init(config *authority.Config) (*CA, error) {
 		routerHandler.Route(r)
 	})
 
-	if config.ACME != nil {
-		//Add ACME api endpoints in /acme and /1.0/acme
-		dns := config.DNSNames[0]
-		u, err := url.Parse("https://" + config.Address)
-		if err != nil {
-			return nil, err
-		}
-		port := u.Port()
-		if port != "" && port != "443" {
-			dns = fmt.Sprintf("%s:%s", dns, port)
-		}
-		prefix := config.ACME.Prefix
-		acmeAuth := acme.NewAuthority(auth.GetDatabase().(nosql.DB), dns, prefix, auth)
-		acmeRouterHandler := acmeAPI.New(acmeAuth)
-		mux.Route("/"+prefix, func(r chi.Router) {
-			acmeRouterHandler.Route(r)
-		})
-		mux.Route("/1.0/"+prefix, func(r chi.Router) {
-			acmeRouterHandler.Route(r)
-		})
+	//Add ACME api endpoints in /acme and /1.0/acme
+	dns := config.DNSNames[0]
+	u, err := url.Parse("https://" + config.Address)
+	if err != nil {
+		return nil, err
 	}
+	port := u.Port()
+	if port != "" && port != "443" {
+		dns = fmt.Sprintf("%s:%s", dns, port)
+	}
+
+	prefix := "acme"
+	acmeAuth := acme.NewAuthority(auth.GetDatabase().(nosql.DB), dns, prefix, auth)
```

Comment:
We should think if it makes sense to add support for multiple DNS entries and always use the one used by the user in the initialization (account creation?)